data/docs/metrics-management/docker-container-metrics.mdx

```diff
@@ -0,0 +1,199 @@
+---
+date: 2024-12-08
+title: Docker container metrics
+id: docker-container-metrics
+---
+
+This document provides an explains how to monitor Docker container Metrics using OTel Collector and SigNoz.
+
+## Pre-requisites
+
+- [OTel Collector](/docs/tutorial/opentelemetry-binary-usage-in-virtual-machine/).
```

Comment:
This should be the first step of the document instead of a pre-requisite.

You can remove it from here and create a section just after Pre-requisites

## Setup OTel Collector